chapters/synchronous.tex

```diff
@@ -1265,6 +1268,58 @@ \subsection{Solver Methods}\label{solver-methods}
 This data is tool specific and is typically either defined with a vendor annotation or is given in the simulation environment.
 \end{nonnormative}
 
+\subsection{Setting states}\label{setting-states}
+\begin{nonnormative}
+In order to handle measurement in control systems it is necessary to consistently update the discretized states, see \textcite{OlssonEtAl2017ModelBased}.
+This replaces the previous discretization methods for specific variables.
+\end{nonnormative}
+
+A declaration equation can have the form \lstinline!Real x=reinit(xs);!
+The restrictions are that:
+\begin{itemize}
+\item It is only legal in a Discretized Continuous-Time partition.
+\item The variable (i.e.,\ \lstinline!x!) must be a scalar or array variable that is a subtype of Real.
+\item The right-hand-side must be solely consist of a call to \lstinline!reinit!.
+\item The argument to \lstinline!reinit! (i.e,\ \lstinline!xs!) must be compatible with the variable, and may be an expression.
+\end{itemize}
+
+The semantics are:
+\begin{itemize}
+\item The variable implicitly always has \lstinline!StateSelect.always!.
+\item The variable participate in the usual index reduction and state selection (where it must be selected as a state).
+\item The discretization of the state is equal to the new value \lstinline!xs! during the entire step, instead of using the discretization method in \cref{solver-methods}.
+\item This declaration equation is ignored for the equation count.
```

Comment:
Add cross reference:
```suggestion
\item This declaration equation is ignored for the equation count, see \cref{local equation size}.
```

Additionally, that definition should be updated to not count `reinit` declaration equations.